ambry-store/src/main/java/com.github.ambry.store/InMemoryIndex.java

```diff
@@ -1,557 +0,0 @@
-/**
```

Comment:
I don't remember on top of my head. But any idea as to why this class was never used? Or was the usages was removed due to some dead code removal or as part of some other work.